source/grid/tria.cc

```diff
@@ -1689,7 +1689,7 @@ namespace internal
               default:
                 // in 1d, a node must have one or two adjacent lines
                 if (spacedim==1)
-                  AssertThrow (false, ExcInternalError())
+                  Assert (false, ExcInternalError())
```

Comment:
I have seen that in a couple of place in the library. I assume it's ok because there is a semicolon at the end of the else...